proxies/dotnet/dotnet.csproj

```diff
@@ -6,6 +6,8 @@
     <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(RunConfiguration)' == 'https' " />

```

Comment:
is this to fix the https redirect warning? is it gonna redirect all requests to https after this change? not sure that will work in docker

Reply:
i'm not sure what this does tbh, it was auto generated when i ran it in vs code so i assumed it was needed to run dotnet run directly - but it works fine without it so i'll remove